omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@nimid I mean, we can separate the responsibility of classes.
Since `OmisePaymentModuleFrontController` is a class that communicate between Front & Back. This class knows, what items are in a cart, which currency buyer's using, total amount of a cart.

And `Charge` knows Omise's format.

Then, code layer could be
```
       |                           |
       |                           |
Omise < > Charge                  < > OmisePaymentModuleFrontController
=======================================================================
       | • Formatting cart's info  | • Collect cart's informations
       |   to Omise format         | • Pass params to Charge
       | • OmiseCharge::create();  | • Charge::create(informations);
       |                           | 
```